packages/vue-pdf-viewer/src/components/PDFViewer.vue

```diff
@@ -127,22 +102,7 @@ export default {
             mdiCropFree,
             pdfViewer: null,
             pdfDocument: null,
-            scaleOptions: [
-                { label: this._$t('pdfViewer.scale.auto'), value: 'auto' },
-                { label: this._$t('pdfViewer.scale.pageActual'), value: 'page-actual' },
-                { label: this._$t('pdfViewer.scale.pageFit'), value: 'page-fit' },
-                { label: this._$t('pdfViewer.scale.pageWidth'), value: 'page-width' },
```

Comment:
Remove these language variables also from the locale files if they are not needed anymore.